phial-scope/src/main/java/com/mindcoders/phial/internal/PhialScopeNotifier.java

```diff
@@ -26,13 +26,13 @@ static void removeListener(OnScopeChangedListener listener) {
         LISTENERS.remove(listener);
     }
 
-    protected static void fireEnterScope(String scope) {
+    protected static void fireEnterScope(Scope scope) {
```

Comment:
better to change to `fireEnterScope(String name, View view)` to keep api simpler.